[Content] Remnant Job To Disable Worldship (#7698)
endless-sky/endless-sky: data/remnant/remnant 2 side missions.txt

			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
			label scanningpayment
			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

> 

Reviewer: ```suggestion

```
3 spaces between the missions

mission "Remnant: Wont Someone Please Think Of The Children"
	landing
	name "Sympathy For The Korath"

> 	description "Travel to the <waypoints> system to fight and disable a stranded Ra'gru Ik 618. Do not destroy or capture the ship. Our intel says the world ship has a large civilian population."

Reviewer 2: Wanderers refer to these as world-ships because they don't have much experience with Korath language and culture.  The Remnant should know better.  Also, they don't need intel to know a Korath ship that big is going to be carrying lots of passengers.

```suggestion
	description "Travel to the <waypoints> system to fight and disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
```

	source
		government "Remnant"
		not attributes "station"
	waypoint "Parca"
	to offer
		has "event: remnant: cognizance calm"
		"combat rating" > 8103
	on offer
		conversation

> 			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request of you."`

Reviewer: ```suggestion
			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request for you."`
```
alternatively, to ask of you?

> 			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Ra'gru Ik 618 to be precise. Normally we would request you track down this ship and eliminate, but our scouts also reported that this appears to be home to a large civilian population."`

Reviewer 2: ```suggestion
			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Kas'lor Ik 582 to be precise. Normally we would request you track down this ship and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```

> 			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can arranged for a prisoner swap with the Korath."`

Reviewer: Now, whilst I like the idea of it, I feel like remnant - Korath interactions are based on things that are not said, not explicit arrangements, although they both don't want to harm the other too much, and will try to find the best ways to approach things. So, I'd make it more like this:
```suggestion
			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we will find a way to bring them home."`
```
But don't change this until Zitchas gets to look at it

Reviewer 2: I think this is more in keeping with the way Zitchas envisions these encounters?

```suggestion
			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can arrange to have the ship disarmed and sent back to a Korath station."`
```

Reviewer: Thats the thing, how do you bring it back? You fix the JD? Give it another one? That's why I kept it vague

Reviewer 2: Oh, right, I hadn't thought of *that* aspect.  This will require some thought when I've not stayed up through the night again.

Author: Added a requirement that you have completed a total of at least 10 Broken Jump drive missions, and changed this that the Remnant will disarm the ship and then fix their jump drive, so the Korath will hopefully decide to just go home.

			choice
				`	"Happy to help those in need, even the Korath."`

> 				`	"I have intention of risking my neck for some Korath."`

Reviewer: ```suggestion
				`	"I have no intentions of risking my neck for some Korath."`
```

					goto rathernot

> 			`	"I am grateful for your understanding and willingness to help here. Lets make sure everyone gets home safely."`

Reviewer: ```suggestion
			`	"I am grateful for your understanding and willingness to help here. Let's make sure everyone gets home safely."`
```

				accept
			label rathernot
			`	He nods in acknowledgment. "I feared my ask was too much. I more than understand the sentiment. Maybe I was being naive to think such an outcome was possible."`
				decline
	npc disable save
		government "Korath"
		personality heroic vindictive target marked staying
		system "Parca"

> 		ship "Ra'gru Ik 618 (Crippled)" "Flumgorkor"

Reviewer 2: ```suggestion
		ship "Kas'lor Ik 582 (Crippled)" "Flumgorkor"
```

	on enter "Parca"
		dialog

> 			`As you enter the sector your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of the children.`

Reviewer: ```suggestion
			`As you enter the sector your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of the defenseless Koraths on board, possibly even children.`
```

Reviewer 2: Korath civilians are... not defenseless >:-)

Also the plural is Korath not Koraths.

```suggestion
			`As you enter the system your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of civilians on board, possibly even children.`
```

Reviewer: Its a 618, the most military oriented vessel out of all world ships, surely these are not civilians, or they wouldnt have strayed away?

	on complete
		payment 2500000
		conversation
			`You find Chilia with a relieved look on his face when you land.`

> 			`	"The prisoner swap wasn't my idea. I was about ready to fill out the paperwork for the official bounty when one our scouts came to my office to make a personal appeal."`

Reviewer: Paperwork? idk honestly, but same Zitchas should look at it

Reviewer 2: Yeah this whole bit is a bit off.  After hundreds of years of fighting Korath, I expect the Remnant know all this by now.  It's also a bit extreme.

Author: Update with @ravenshining Suggestion resolving this conversation

			`	"She explained that on her last cloaked fly by scanning run of the 618 she noticed what looked like large group of Korath children playing some type of ball game on the thermal scan."`

> 			`	"When she landed she went and looked at the biometric scan data and realized the scan showed around 800 uniq bio signatures, but with only about 7/8th the expected bio mass. Meaning 1/4th of the ships passengers could be children."`

Reviewer: ```suggestion
			`	"When she landed she went and looked at the biometric scan data and realized the scan showed around 800 unique bio signatures, but with only about 7/8th the expected bio mass. Meaning 1/4th of the ships passengers could be children."`
```

> 			`	"The prisoner swap wasn't my idea. I was about ready to fill out the paperwork for the official bounty when one our scouts came to my office to make a personal appeal."`
> 			`	"She explained that on her last cloaked fly by scanning run of the 618 she noticed what looked like large group of Korath children playing some type of ball game on the thermal scan."`
> 			`	"When she landed she went and looked at the biometric scan data and realized the scan showed around 800 uniq bio signatures, but with only about 7/8th the expected bio mass. Meaning 1/4th of the ships passengers could be children."`
> 			`	"I figured you where those children's best chance. I am glad I was not wrong." he says with a content smile on his face.`
> 			`	"I worry in our ignorance of Korath society we may have unnecessarily destroyed other ships like this one..."`
> 			`	"Expect more missions like this in the future." Chilia then walks away invigorated with new purpose.`

Reviewer 2: ```suggestion
			`	"This mission wasn't my idea. Taking on the larger Korath vessels is a sensitive matter. You may frequently find large groups of Korath children. I must admit I was worried you wouldn't be those children's best chance, but our fleet was too tied up to take this one at the moment. I am glad I was not wrong." he says with a content smile on his face.`
			`	"We must be careful not to let ignorance of Korath society result in the unnecessary destruction of similar ships. Now that you have shown you can handle these bounties with the proper restraint, expect more missions like this in the future." Chilia then walks away invigorated with new purpose.`
```

Author: Updated with your suggested text, but couldn't use the commit button because of other changes I had already accepted.

			`	"Would you be up for disabling this Korath Kas'lor Ik 582, so the threat is eliminated while preserving the lives of the innocents on board?" His signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
			choice
				`	"Happy to help those in need, even the Korath."`
				`	"I have no intentions of risking my neck for some Korath."`
					goto rathernot
			`	"I am grateful for your understanding and willingness to help here. Together we can ensure everyone gets home safely."`
				accept
			label rathernot
			`	He nods in acknowledgment and his gestures take on a staccato phrasing. "I feared my ask was too much. I understand the sentiment. Perhaps I was naive to think such an outcome was possible."`
				decline
	npc disable save
		government "Korath"

> 		personality heroic vindictive target marked staying

Reviewer: why heroic? why not timid?

> 		personality heroic vindictive target marked staying

Reviewer 2: Consider what they're defending.  A ship with many civilians on board, equipped with long-range artillery?  The `olofezes I think should do whatever it takes.... especially since they can't exactly flee anywhere.

> 		personality heroic vindictive target marked staying

Reviewer: timid means they stay near the ship and thus are **much** more efficient at defending it

> 		personality heroic vindictive target marked staying

Reviewer: it seems to work
heroic is the worst thing to give to a fighter it needs to stay near the flagship to get repairs and actually defend it

> 		personality heroic vindictive target marked staying

Reviewer 2: Cool.  Just mind you this is the mothership not the fighters, I'll change that on them.  They have swarming to keep them near the ship, too.

> 		personality heroic vindictive target marked staying

Reviewer: I did tell you that I thought swarming ships shoot their own? Anyhow if that's still the case its a separate thing that needs fixing

		system "Vaticanus"

> 		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"

Reviewer: best to do this in the ship variant definition if we want it

> 		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"

Reviewer 2: We could, although I like the idea of making the first mission harder, and having the jobs be more variable.

			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Kas'lor Ik 582 to be precise. Normally we would request you track down this ship and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can disarm the ship and fix their jump drive and hope they decide to head back home."`
			choice
				`	"Happy to help those in need, even the Korath."`
				`	"I have no intentions of risking my neck for some Korath."`
					goto rathernot
			`	"I am grateful for your understanding and willingness to help here. Let's make sure everyone gets home safely."`
				accept
			label rathernot
			`	He nods in acknowledgment. "I feared my ask was too much. I more than understand the sentiment. Maybe I was being naive to think such an outcome was possible."`
				decline
	npc disable save
		government "Korath"
		personality heroic vindictive target marked staying
		system "Antevorta"

> 		ship "Kas'lor Ik 582 (Crippled)" "Flumgorkor"

Author: Just as context I took three of the name parts from the Korath definition file. So hopefully I understood how the name system works.

> 		ship "Kas'lor Ik 582 (Crippled)" "Flumgorkor"

Reviewer 2: Are you sure you were looking at the right file?  Because I can't find bits that would make that up.  If I refer to an older version of the file, however, this can be done:

```suggestion
		ship "Kas'lor Ik 582 (Crippled)" "Fligrookor"
```

		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
	on enter "Antevorta"
		dialog
			`As you enter the system your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of civilians on board, possibly even children.`
	on abort
		"reputation: Remnant" += 250
	on fail
		"reputation: Remnant" -= 250
		dialog
			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
	on complete
		"remnant: disable and save count" ++
		payment 2500000
		conversation
			`You find Chilia with a relieved look on his face when you land.`

> 			`	"This mission wasn't my idea. Taking on the larger Korath vessels is a sensitive matter. You may frequently find large groups of Korath children. I must admit I was worried you wouldn't be those children's best chance, but our fleet was too tied up to take this one at the moment. I am glad I was not wrong." he says with a content smile on his face.`

Reviewer 3: ```suggestion
			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matte. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessment of you." he sings with a content smile on his face.`
```

It is subtle, but shifting the concerns from Chilia specifically to the Remnant in general is intentional to suggest and reinforce the idea that there is a significant collaborative decision making process behind the scenes.

> 			`	"We must be careful not to let ignorance of Korath society result in the unnecessary destruction of similar ships. Now that you have shown you can handle these bounties with the proper restraint, expect more missions like this in the future." Chilia then walks away invigorated with new purpose.`

Reviewer 3: ```suggestion
			`	"We must be careful not to let ignorance of Korath society result in the unnecessary destruction of similar ships. Now that you have shown you can handle these tasks with the proper finesse, you can expect these to turn up in the task board in the future." Chilia then walks away with a confident pace.`
```

			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
			label scanningpayment
			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

mission "Remnant: Will Not Someone Please Think Of The Children"
	landing
	name "Sympathy For The Korath"
	description "Travel to the <waypoints> system to disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
	minor
	source
		government "Remnant"

> 		not attributes "station"

Reviewer 4: I don't think an object exists with the Remnant government and the `station` attribute.
Ssil Vida has the `remnant station` attribute, but it seems like the desire is for this mission to be available on Ssil Vida after a certain point.

Author: Chilia on Ssil Vida seemed weird to me. 

Also I was actually kind of worried about all the sector modification logic that happens with Ssil Vida. So a little bit of fear of the unknown also and reduces possible future issues.

Reviewer 4: The current `to offer` conditions say to me this mission should have the possibility of being offered on Ssil Vida from the end of the Remnant chapter 2 story onwards. If this mission should not be offered there, then the `to offer` conditions should be simplified and this location filter should be changed to `not attributes "remnant station"` since that's the attribute Ssil Vida has.
The attributes of Ssil Vida are not changes at any point in the existing events.
Regardless, `not attributes "station"` is just confusing since its presence doesn't do anything unless I've missed a planet.

Author: Changed to allow to be offered on Ssil Vida, only source requirement now is Remnant government.

			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board?" his signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
			choice
				`	"Happy to help those in need, even the Korath."`
				`	"I have no intentions of risking my neck for some Korath."`
					goto rathernot
			`	"I am grateful for your understanding and willingness to help here. Together we can ensure everyone gets home safely."`
				accept
			label rathernot
			`	He nods in acknowledgment and his gestures take on a staccato phrasing. "I feared my ask was too much. I understand the sentiment. Perhaps I was naive to think such an outcome was possible."`
				decline
	npc disable save
		government "Korath"
		personality heroic vindictive target marked staying

> 		system "Antevorta"

Reviewer 4: The conversation says `Parca`, not `Antevorta`.
Also, how is this ship in Antevorta? That's not a trivial journey from the Korath Exile systems, especially not without a keystone of some sort (which the Exiles are either supposed to not have or not know how to use).
Is the idea that this ship had a key stone or similar but lost it?

Author: It was originally in Parca, but moved it to  Antevorta in response to this comment

https://github.com/endless-sky/endless-sky/pull/7698#issuecomment-1326566610

Fixing the text not being updated in all places though

Reviewer: Yeah no exile keystones but mb it was the drive malfunctionning?

Author: Yeah will update the variant to stranded to make it clearer as to what happened.

Reviewer 4: The drive malfunctioning allowing passage through the wormhole is certainly something.

Reviewer: More like it teleported it far away and then broke

Reviewer: But could be best to be in Parca still

Reviewer 4: Honestly, it would probably make most sense for it to be trapped in the Postverta cluster.

Author: IE When you toggle the clusters state via the job "[De]Activate Ssil Vida Power"

Author: NVM just found the events, it looks like it updates the systems and not replaces them. Can change to that cluster then. Would make more sense.

https://github.com/endless-sky/endless-sky/blob/master/data/remnant/remnant%20events.txt#L344-L1132

Reviewer: Maybe it should be stranded near Stercutus?

Author: Changed to Postverta cluster and added Ssil Vida as a spawn point for the side mission.

		ship "Kas'lor Ik 582 (Crippled)" "Flumgorkor"
		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
	on enter "Antevorta"
		dialog

> 			`As you enter the system your scanners pick the target ship, and reveal an impressive amount of weaponry and military hardware. The environmental data reveals a small city's worth of people, many whose size indicates they are probably children.`

Reviewer 2: It ain't THAT big

```suggestion
			`As you enter the system your scanners pick the target ship, and reveal an impressive amount of weaponry and military hardware. The environmental data reveals a small town's worth of people, many whose size indicates they are probably children.`
```

	on abort
		"reputation: Remnant" += 250
	on fail
		"reputation: Remnant" -= 250

> 		dialog
> 			`You were unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`

Reviewer: ```suggestion
		conversation
			branch failed
				not "Remnant: Will Not Someone Please Think Of The Children: aborted"
			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They warn you they will not propose any more of the sort to you again.`
				decline
			label failed
			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```

> 			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`

Reviewer 2: ```suggestion
			`You were unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```

			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
			label scanningpayment
			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

mission "Remnant: Will Not Someone Please Think Of The Children"
	landing
	name "Sympathy For The Korath"
	description "Travel to the <waypoints> system to disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."

> 	minor

Reviewer: Its before

			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board?" his signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
			choice
				`	"Happy to help those in need, even the Korath."`
				`	"I have no intentions of risking my neck for some Korath."`
					goto rathernot
			`	"I am grateful for your understanding and willingness to help here. Together we can ensure everyone gets home safely."`
				accept
			label rathernot
			`	He nods in acknowledgment and his gestures take on a staccato phrasing. "I feared my ask was too much. I understand the sentiment. Perhaps I was naive to think such an outcome was possible."`
				decline
	npc disable save
		government "Korath"
		personality heroic vindictive target marked staying
		system "Vaticanus"
		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"
		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
	on enter "Vaticanus"
		dialog

> 			`As you enter the system your scanners pick the target ship, and reveal an impressive amount of weaponry and military hardware. The environmental data reveals a small town's worth of people, many whose size indicates they are probably children.`

Author: Changing this up to remove reference to sensors

	on abort
		"reputation: Remnant" += 250
	on fail
		"reputation: Remnant" -= 250
		conversation
			branch failed
				not "Remnant: Will Not Someone Please Think Of The Children: aborted"
			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They warn you they will not propose any more of the sort to you again.`
				decline
			label failed
			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
	on complete
		"remnant: disable and save count" ++
		payment 2500000
		conversation
			`You find Chilia with a relieved look on his face when you land.`

> 			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matter. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessment of you." he sings with a content smile on his face.`

Reviewer 4: Capital after the speech ends with a full stop?
Also, did you definitely mean `sings` or was it supposed to be `signs`?

			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
			label scanningpayment
			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

mission "Remnant: Will Not Someone Please Think Of The Children"
	minor
	landing
	name "Sympathy For The Korath"
	description "Travel to the <waypoints> system to disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
	source
		government "Remnant"
	waypoint "Vaticanus"
	to offer
		random < 30
		"remnant: broken jump drive count" >= 10
		or

> 			and
> 				has "Remnant: Cognizance 25: done"
> 				not "flagship planet: Ssil Vida"
> 			has "event: remnant: cognizance calm"

Reviewer 5: ```suggestion
			has "event: remnant: cognizance calm"
			and
				has "Remnant: Cognizance 25: done"
				not "flagship planet: Ssil Vida"
```

	on offer
		conversation

> 			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you."`

Reviewer 5: ```suggestion
			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you.`
```

> 			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`

Reviewer 6: ```suggestion
			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations, we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```

> 			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations, we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`

Reviewer 5: ```suggestion
			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations, we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector.`
```

> 			`	"Our scouts are reporting a stranded Korath Kas'lor Ik 582 in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`

Reviewer 4: `a stranded Korath Kas'lor Ik 582 in Vaticanus, a Kas'lor Ik 582 to be precise.`

I think something got messed up in a previous suggestion.

```suggestion
			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```

There should perhaps also be a comma between `operations` and `we`, but I'm not sure. I would put one there but that might just be my writing style. @EjoThims thoughts?

> 			`	"Would you be up for disabling this Korath Kas'lor Ik 582, so the threat is eliminated while preserving the lives of the innocents on board?" His signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`

Reviewer 4: Calm down, Chilia, you don't need to tell me it's a Korath ship and give me the full model name every time, Man is obsessed with saying this.
```suggestion
			`	"Would you be up for disabling it, so the threat is eliminated while preserving the lives of the innocents on board?" His signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
```

			choice
				`	"Happy to help those in need, even the Korath."`
				`	"I have no intentions of risking my neck for some Korath."`
					goto rathernot
			`	"I am grateful for your understanding and willingness to help here. Together we can ensure everyone gets home safely."`
				accept
			label rathernot
			`	He nods in acknowledgment and his gestures take on a staccato phrasing. "I feared my ask was too much. I understand the sentiment. Perhaps I was naive to think such an outcome was possible."`
				decline
	npc disable save
		government "Korath"
		personality heroic target marked staying disables plunders
		system "Vaticanus"
		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"

> 				crew 551

Reviewer 5: This still doesn't work.
```suggestion
```

> 		dialog `You have disabled the Fligrookor, time to report back to <planet>.`

Reviewer 5: The job just uses the model name, so might as well keep it consistent.
```suggestion
		dialog `You have disabled the Kas'lor Ik 582. Time to report back to <planet>.`
```

	npc
		government "Korath"
		personality timid disables plunders swarming
		system "Vaticanus"
		fleet
			names "korath"
			variant
				"'olofez (Digger)" 3
	on enter "Vaticanus"
		dialog `As you enter the system you spot the target ship. You imagine the small town's worth of people inside, many of which are probably children.`
	on abort
		dialog `The Remnant are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They warn you they will not propose any more of the sort to you again.`
	on fail
		"reputation: Remnant" -= 250
		dialog `You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
	on complete
		"remnant: disable and save count" ++
		payment 2500000
		conversation
			`You find Chilia with a relieved look on his face when you land.`

> 			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matter. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessment of you." he signs with a content smile on his face.`

Reviewer 5: ```suggestion
			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matter. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessment of you." He signs with a content smile on his face.`
```
